buildSrc/src/main/kotlin/CodegenTestCommon.kt

```diff
@@ -179,7 +182,7 @@ fun Project.registerGenerateCargoConfigTomlTask(
     this.tasks.register("generateCargoConfigToml") {
         description = "generate `.cargo/config.toml`"
         doFirst {
-            outputDir.resolve(".cargo").mkdir()
+            outputDir.resolve(".cargo").mkdirs()
```

Comment:
Was this just broken before? What's the difference?

Reply:
It was breaking for me. I'm not sure how it worked for anyone unless the parent directories happened to exist.